src/app/zap-templates/zcl/data-model/chip/dishwasher-mode-select-cluster.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="SemanticTags" type="ENUM16">
+    <cluster code="0x0059"/>
+    <item value="0x4000" name="Normal"/>
+    <item value="0x4001" name="Heavy"/>
+    <item value="0x4002" name="Light"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Dishwasher Mode Select</name>
+    <code>0x0059</code>
+    <define>DISHWASHER_MODE_SELECT_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster is an alias of the Mode Select cluster, defining additional semantics and
+      namespaced enumerated values for dishwasher devices.</description>
+    <!-- Base data types -->
+    <attribute side="server" code="0x0000" define="MODE_DESCRIPTION" type="CHAR_STRING"                                     writable="false" optional="false" isNullable="false" length="64">Description</attribute>
+    <attribute side="server" code="0x0001" define="STANDARD_NAMESPACE" type="ENUM16"                                        writable="false" optional="false"  isNullable="true">StandardNamespace</attribute>
```

Comment:
Also, why are we adding this deprecated attribute to the new aliases, and adding it as a mandatory attribute?

Reply:
Good spot, I did not notice this change. Accepted.